drivers/mtd/nand/raw/pl35x-nand-controller.c

```diff
@@ -795,6 +795,22 @@ static int pl35x_nfc_exec_op(struct nand_chip *chip,
 				      op, check_only);
 }
 
+static unsigned long get_special_timing_freq(const struct device_node *np, const int timing_mode)
+{
+	int len, i;
+
+	const __be32 *memclk_timing_freq = of_get_property(np, "memclk-timing-frequency", &len);
+
+	if (memclk_timing_freq) {
+		for (i = 0; i < len / sizeof(u32) / 2; i++) {
+			if (timing_mode == be32_to_cpu(memclk_timing_freq[i * 2])) {
+				return be32_to_cpu(memclk_timing_freq[i * 2 + 1]);
+			};
```

Comment:
Extra ';' and unnecessary '}'. See the section on "3) Placing Braces and Spaces" of the kernel's coding style document: https://www.kernel.org/doc/html/latest/process/coding-style.html

Specifically "Do not unnecessarily use braces where a single statement will do."